client/src/BidderBiddingList.js

```diff
@@ -0,0 +1,62 @@
+
+
+import { useEffect, useState } from "react";
+
+const BidderBiddingList = () => {
+    const [loading, setLoading] = useState(true);
+    const [bidderList, setBidderList] = useState([]);
+    const [errorMsg, setErrorMsg] = useState(null);
+    const [page, setPage] = useState(1);
+
+    function dateFormat(date) {
+        return date.split("T")[0];
+    }
+
+    useEffect(() => {
+        const fetchBidderBids = async () => {
+            try {
+                const response = await fetch(`api/bidder-bid?page=${page}`);
+                if (!response.ok) {
+                    throw new Error("Problem with the server!");
+                }
+                const data = await response.json();
+                setLoading(false);
+                setBidderList(data);
+            } catch (error) {
+                setErrorMsg(error.message);
+            }
+        };
+        fetchBidderBids();
+    }, [page]);
+
+    if (errorMsg !== null) {
+        return <div>{errorMsg}</div>;
+    }
+
+    if (loading) {
+        return <div>Loading!!</div>;
+    }
+
+    return (
+        <>
+            <h1>Buyer Tenders List</h1>
+            <div className="bids-container"> {bidderList.map((bid, index) =>
+                <div className="bid-card" key={index}>
+                    <p>Status: {bid.status}</p>
+                    <p>submitted on: {dateFormat(bid.bidding_date)}</p>
+                    <p>Bidding Amount: {bid.bidding_amount}</p>
+                    <div>Cover Letter:
+                        <p>{bid.cover_letter}</p>
+                    </div>
+                    <p>Completion Time: {bid.suggested_duration_days} days</p>
+                </div>
+            )}
+                <button className={bidderList.length < 25 ? "show" : "hide"} onClick={() => setPage((prev) => prev - 1)}>Back</button>
```

Comment:
There is no relationship between the size of the list and the existence of a previous page. This logic is wrong.

If you want, you can show the button all the time for now, since we will have to standardize pagination as I mentioned on the web wizards Slack channel.